CallCenterSample/Helpers/TextAnalyticsHelper.cs

```diff
@@ -109,48 +123,41 @@ public static async Task<DetectLanguageResult> GetDetectedLanguageAsync(string i
 
             if (!string.IsNullOrEmpty(input))
             {
-                LanguageBatchResult result = await AnalyticsClient.DetectLanguageAsync(new BatchInput(
-                    new List<Input>()
-                    {
-                        new Input("0", input)
-                    }));
-
-                if (result.Documents != null)
-                {
-                    languageResult.Language.Add("iso6391Name", result.Documents[0].DetectedLanguages[0].Iso6391Name);
-                    languageResult.Language.Add("name", result.Documents[0].DetectedLanguages[0].Name);
-                    languageResult.Language.Add("score", result.Documents[0].DetectedLanguages[0].Score.ToString());
-                }
+                DetectLanguageResultCollection result = await AnalyticsClient.DetectLanguageBatchAsync(
+                new List<string>() { input });
 
-                if (result.Errors != null)
+                if (!result[0].HasError)
                 {
-                    // Just return the empty Dictionary
-                }
+                    languageResult.Language.Add("iso6391Name", result[0].PrimaryLanguage.Iso6391Name);
+                    languageResult.Language.Add("name", result[0].PrimaryLanguage.Name);
+                    languageResult.Language.Add("score", result[0].PrimaryLanguage.ConfidenceScore.ToString());
+                }   
             }
 
             return languageResult;
         }
 
         public static async Task<SentimentResult> GetTextSentimentAsync(string input, string language = "en")
         {
-            SentimentResult sentimentResult = new SentimentResult() { Score = 0.5 };
+            SentimentResult sentimentResult = new SentimentResult()
+            {
+                Sentiment = "null",
+                Negative = 0.5,
+                Neutral = 0.5,
+                Positive = 0.5
```

Comment:
if this is trying to mimic how the TA scores work, this is misleading. All the scores need to add up to 1 as they represent the Softmax for each sentiment label.
Also, what will `Sentiment` = null mean?

Reply:
I understand that normally they should add up to 1.
These are initial values (including `Sentiment = "null"`), they will be returned only when sentiment analyzing fails or the input text is empty.
I want this to stand for an empty/pointless analysis result.